src/main/java/ua/ithillel/lesson4/ArrayValueCalculator.java

```diff
@@ -0,0 +1,29 @@
+package ua.ithillel.lesson4;
+
+public class ArrayValueCalculator {
+    public static final int SIZE_ARRAY = 4;
+
+    public int doCalc(String[][] myArray) throws ArraySizeException, ArrayDataException {
```

Comment:
```suggestion
    public int doCalc(String[][] myArray)  {
```
вони у тебя unchecked, тому не треба їх вказувати.